pycaching/geocaching.py

```diff
@@ -382,6 +382,27 @@ def _cache_from_guid(self, guid):
         print_page = self._request(Cache._urls["print_page"], params={"guid": guid})
         return Cache._from_print_page(self, guid, print_page)
 
+    def _try_getting_cache_from_guid(self, guid):
+        """tries to read a geocache from GUID page
```

Comment:
Please follow the same format for docstrings as the other functions. This is important because we use it to generate docs.